src/app/aircraft/add/post.controller.js

```diff
@@ -48,12 +49,16 @@ module.exports = (req, res) => {
             if (_.toString(apiResponse).includes('DETAIL:  Key (registration)')) {
               errMsg = { message: 'Craft already exists' };
             }
-            res.render('app/aircraft/add/index', { cookie, errors: [errMsg] });
+            res.render('app/aircraft/add/index', {
+              cookie, craftType, craftReg, craftBase, errors: [errMsg],
+            });
           }
         });
     })
     .catch((err) => {
       logger.info('Add craft postcontroller - There was a problem with adding the saved craft');
-      res.render('app/aircraft/add/index', { cookie, errors: err });
+      res.render('app/aircraft/add/index', {
+        cookie, craftType, craftReg, craftBase, errors: err,
```

Comment:
Return the entered values otherwise on validation, the input fields are cleared, which could frustrate users.